packages/esm-patient-tests-app/src/test-results/grouped-timeline/grouped-timeline.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import GroupedTimeline from './grouped-timeline.component';
+import { mockGroupedResults } from '../../../../../__mocks__/grouped-results.mock';
+import FilterContext from '../filter/filter-context';
+import { type FilterContextProps } from '../filter/filter-types';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({ t: (key: string) => key }),
+}));
```

Comment:
We don't need to mock this again. It already gets mocked [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/jest.config.js#L19).